client/state/selectors/get-locale-suggestions.js

```diff
@@ -0,0 +1,15 @@
+/** @format */
+/**
+ * External Dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns an object of localized language names
+ *
+ * @param  {Object}  state Global state tree
+ * @returns {Array|Null} an array of guessed locales for the user
+ */
+export default function getLocaleSuggestions( state ) {
+	return get( state, 'i18n.localeSuggestions', null );
```

Comment:
since it's not clear to me how this will be used I'm not sure if this advice is useful, but what does `null` mean? if we're wanting to check "have we loaded actual suggestions? if so then use them" then this is good. but, if we want to simplify logic in places and instead ask, "what are the current suggestions?" then we can also return an empty list by default. the empty list is a safe and valuable way to indicate the absence of information. we may not know if it's because it hasn't loaded or if it's because there are no suggestions, but once it _does_ load that list will fill up.

Reply:
> if we're wanting to check "have we loaded actual suggestions? 

This was the original intention. 

For the time being, this HTTP call occurs only in `<LocaleSuggestions />`.

[LocaleSuggestions](https://github.com/Automattic/wp-calypso/blob/3c96f6f/client/components/locale-suggestions/index.jsx) performs a filter/map, but we could always return an empty array by default, and then do an `isEmpty` or length check before we make the component do any work. But the `null` return was just a cheap way of knowing that a return value is pending.

This PR is a stepping stone to some work which will update `<LocaleSuggestions />` anyway.